Templates/project_collection/README.md

```diff
@@ -0,0 +1,212 @@
+  * A landing page with a general overview of your project is best
+  * If you don't have a landing page, you could include a DOI for a flagship paper,
+    or a link to the repository to where the data for this project are stored
+  
+#### **is funded by** - *grant*
+  * This is the grant that is funds your project
+  * This field is controlled by the vocabulary in the `grant` column on `values` sheet
+  * If you don't see your grant on that list, please email 
+    [data.curation@alleninstitute.org](mailto:data.curation@alleninstitute.org), 
+    and we will add it
+
+#### **has use of results limited by** - *license*
+  * This is the usage license for the data collected by this project
+  * This field is controlled by the vocabulary in the `license` column on `values` sheet
+  * For most BICCN projects this should be `CCBY4` (https://creativecommons.org/licenses/by/4.0/)
+  * If the project collected Allen Institute data it might use `AIBSTOU` instead
+  * Please contact data.curation@alleninstitute.org if you are not sure what license to use.
+
+#### **has methodology specification** - *protocol URL*
+  * This is a hyperlink to a protocol, usually from https://www.protocols.io/.
+  * You can submit multiple values for this field, separated by commas or 
+    new rows immediately after this one
+  * Please make sure the protocol has been published before providing us a link
+  * Provide the URL in the form of a DOI, so we can be consistent with other projects.
+  * Example: `http://doi.org/<doi_number>/protocols.io.<protocol_id>`
+  
+#### **has output** - *collection name*
+  * This is a unique human-readable identifier (i.e. handle) used to keep track 
+    of your data collection
+  * You can submit multiple values for this field, separated by commas or 
+    new rows immediately after this one
+  * This field is controlled by the vocabulary in the `collection` column on `values` sheet
+  * If your project has a new data collection as it's output, please email 
+    [data.curation@alleninstitute.org](mailto:data.curation@alleninstitute.org)
+    and let us know.
+  * Collection names must be unique, so make sure any new data collections do 
+    not have a name that is already in the controlled vocabulary
+  * By convention, we prefer to use names in `snake_case` without `_proj` as a suffix
+  
+#### **has contact person** - *person name*
+  * This is the person that we will send correspondence to if we have questions 
+    or need to talk to you about your project or data collection
+  * You can submit multiple values for this field, separated by commas or 
+    new rows immediately after this one
+  * Usually the contact person is either a data wrangler, or a PI
+  * Please provide names in the following format: `GivenName FamilyName`
+  * `GivenName` can include a middle name(s) or initial(s)
+  * If `FamilyName` has a space in it please let us know
+  
+#### **has contributor** - *person name*
```

Comment:
We are not collecting ORCIDs here yet?

Reply:
Yes, this would be ideal. Currently, if we have two people with the same name way there is no way to handle this or know who is who.